ASI Documentation: User deletes images in bulk from Media Gallery (#626)
magento/merchdocs: src/cms/media-gallery-asset-management.md

    ![]({% link images/images/enhanced-media-gallery/selected-asset.png %}){: .zoom}
    _Add Selected_

## Delete assets

1. On the _Admin_ sidebar, go to **Content** > _Media_ > **Media Gallery**.

> 1. Click <span class="btn">Delete Images...</span> and use checkboxes to select assets that need to be deleted.

Reviewer: ```suggestion
1. Click <span class="btn">Delete Images...</span> and select the checkbox for each asset that you want to delete.
```

>     ![]({% link images/images/enhanced-media-gallery/selected-asset.png %}){: .zoom}

Reviewer: It looks like the PR deletes this file -- did you want to include a replacement?

Author: Yes, it is included.
<img width="1270" alt="Screen Shot 2020-07-28 at 3 37 35 PM" src="https://user-images.githubusercontent.com/17548019/88719079-54af8300-d0e8-11ea-818c-46820be56013.png">
